src/top-navigation/styles.scss

```diff
@@ -230,25 +230,26 @@
   border-block-end: awsui.$border-divider-section-width solid awsui.$color-border-divider-default;
 
   &-text {
-    @include styles.font-panel-header;
     color: awsui.$color-text-heading-default;
     flex: 1;
     margin-block: 0;
     margin-inline: 0;
     text-align: center;
 
+    &--title {
+      /* used in test-utils */
+      @include styles.font-panel-header;
+    }
+
     &--secondary {
       font-size: awsui.$font-header-h2-description-size;
       line-height: awsui.$font-header-h2-description-line-height;
       font-weight: styles.$font-weight-normal;
+      @include styles.font-smoothing;
```

Comment:
This is the only rule that `.overflow-menu-header-text--secondary` really uses from `styles.font-panel-header`, so we add it here explicitly instead of including `styles.font-panel-header` in the parent selector and overriding all its other rules in this child selector.